src/test/java/seedu/fma/storage/JsonLogBookStorageTest.java

```diff
@@ -32,8 +30,8 @@ public void readAddressBook_nullFilePath_throwsNullPointerException() {
         assertThrows(NullPointerException.class, () -> readAddressBook(null));
     }
 
-    private java.util.Optional<ReadOnlyAddressBook> readAddressBook(String filePath) throws Exception {
-        return new JsonAddressBookStorage(Paths.get(filePath)).readAddressBook(addToTestDataPathIfNotNull(filePath));
+    private java.util.Optional<ReadOnlyLogBook> readAddressBook(String filePath) throws Exception {
```

Comment:
Remember to change readAddressBook to readLogBook. This is repeated throughout the file